### DISCUSSION TITLE
Multiline Button using Handlebars

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5821] [Help]
This may be the easiest thing in the world yet i got stuck.
I would like to create a button, with two lines of text. Both lines are coming from handlebar expressions.
How do i set them in the Button's text property two show as two lines ? 

Thank you!

**Comment by melohagan** [comment]
Hey @poirazis 

I wasn't able to find a way to have two lines of text in a button, however I found this handy StackOverflow post: https://stackoverflow.com/questions/862010/how-to-wrap-text-of-html-button-with-fixed-width

You'll also need to set the width of the button to a fixed size. Through trial and error you can get the wrapping to occur on the full-stop: 

![Screenshot 2022-05-11 at 13 02 01](https://user-images.githubusercontent.com/101575380/167845112-99c7e25e-6270-4926-a8ec-7da4eb721797.png)

Out of curiosity, why do you want a button with two lines of text?

**Reply by poirazis** [reply]
I am using it as a hack to create the impression of a clickable container, inside a repeater, so I have access to on Click Actions, and show a modal :) 

<img width="1086" alt="Screenshot 2022-05-11 at 3 21 57 PM" src="https://user-images.githubusercontent.com/80512170/167848842-b95d3a25-db59-474c-9359-84be4807ae36.png">

**Reply by melohagan** [reply]
I've got a PR out for the clickable container: https://github.com/Budibase/budibase/pull/5829
Hopefully it's okay to be merged in at some point!
